sdk/monitor/test-resources.bicep

```diff
@@ -69,6 +71,104 @@ resource metricReaderRole 'Microsoft.Authorization/roleAssignments@2018-01-01-pr
   }
 }
 
+resource table 'Microsoft.OperationalInsights/workspaces/tables@2022-10-01' = {
```

Comment:
Are we the only language that uses Bicep over ARM template? Also, do we have any tools that can say convert an ARM template that we use Internally?

Reply:
Go uses Bicep as well for Monitor Query. I believe this is the internal command but it didn't quite work for me. I used this tutorial page: https://learn.microsoft.com/en-us/training/modules/introduction-to-infrastructure-as-code-using-bicep/5-how-bicep-works. This link to the Deployment Resources page (https://learn.microsoft.com/en-us/azure/templates/microsoft.authorization/roleassignments?pivots=deployment-language-bicep) also helps for a quickly formatted bicep resource deployment.

Reply:
Thanks! Can you point me to the internal command that we use which doesn't work anymore?

Reply:
This is part of the AZ CLI - https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/decompile?tabs=azure-cli#decompile-from-json-to-bicep